sdk/cognitivelanguage/ai-language-text/src/util.ts

```diff
@@ -163,3 +163,24 @@ export function getOperationOptions<OptionsT extends TextAnalysisOperationOption
     rest,
   };
 }
+
+/**
+ *
+ * @param error error with the target in the JSON error pointer format "#/items/0
+ * @returns number: the position of the task with error
+ */
+export function extractErrorPointerIndex(error: ErrorModel): number {
+  if (!error.target) {
+    throw new Error("Error target is not defined to parse the error pointer");
+  }
+  let position: number;
+
+  const listTarget = error.target.split("/");
+  position = parseInt(listTarget.pop() as string);
```

Comment:
Could this be more concisely?
```suggestion
  const position = parseInt(error.target.split("/").pop() as string);
```